src/components/PostCard.jsx

```diff
@@ -174,12 +184,48 @@ const PostCard = ({ post, type }) => {
       </div>
       <div className="flex gap-x-10 gap-y-4 items-center flex-wrap">
         <div className="flex md:gap-3 gap-1 items-center md:flex-row flex-col">
-          <BiLike
-            size={25}
-            className="fill-primary cursor-pointer order-1 md:-order-1"
-          />
+          {!isPostLiked ? (
+            <MdFavoriteBorder
+              onClick={() => {
+                if (auth?.user?._id !== id) {
```

Comment:
you can make auth?.user into a variable and use it as you are accessing it multiple times.

Reply:
Yes, It will be easier to access